core-bundle/src/Resources/contao/library/Contao/DiffRenderer.php

```diff
@@ -99,4 +99,22 @@ public function render()
 
 		return $html;
 	}
+
+	/**
+	 * Converts special characters to HTML entities preserving <ins> and <del> tags
+	 *
+	 * @param string $line
+	 *
+	 * @return string
+	 */
+	private function specialchars($line)
+	{
+		if ($this->options['decodeEntities'])
```

Comment:
I don’t understand how this should work 😕

It seems to me, the only options of the `DiffRenderer` are `tabSize` and `field`. This condition never applies.

Reply:
You can pass custom options in the constructor. 😉 But we will revert the changes anyway.

Reply:
Reverted in 1dfa66c9b2de613fb4f17983c96ff8281d7a8fd2.